Makefile

```diff
@@ -305,3 +305,6 @@ FORCE:
 
 check:
 	./tools/dependency-check.sh
+
+minimal :
+	PYODIDE_PACKAGES="micropip" make
```

Comment:
LGTM, but there is something in your editor configuration that removes newlines at the end of files.  https://unix.stackexchange.com/a/18789

You can [use pre-commit](https://pyodide.readthedocs.io/en/latest/contributing.html#development-workflow) which [would fix it](https://github.com/iodide-project/pyodide/blob/41592637e6a5231d1fa74235ec5f46919c18e25d/.pre-commit-config.yaml#L7).

Reply:
I installed pre-commit, but it always failed so I uninstalled it. Probably should open an issue.

Reply:
Wow I love shit like this: "A file that is not empty and does not end with a newline is therefore not a text file." Very interesting.